models/evc.py

```diff
@@ -159,6 +160,7 @@ def __init__(self, controller, **kwargs):
         self.backup_links_cache = set()
         self.affected_by_link_at = get_time("0001-01-01T00:00:00")
         self.old_path = Path([])
+        self.max_paths = kwargs.get("max_paths", 2)
```

Comment:
This ended up nicely instead of on `PathConstraints` for primary and secondary, that's where we typically should add, but since at the application level `max_paths` (spf_max_paths) has been common for both from `mef_eline` perspective, that's a good compromise and safe bet that will still be used with the same value for both primary and secondary, so let's go for it as you currently have here.